### PR TITLE
Dummy batch API

### PR DESCRIPTION
**Pull request by p1003**

### CODE REVIEW

File: yaptide/routes/batch_routes.py [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                cmd_line_args: <słownik> | <None> (ma wygenerować stringa)
+
+        result not ok:
+            "Error message" (scenariusze):
+                error message od połączenia
+                error message batcha
+
+        result ok:
+            job_id
+        """
```
Comment (grzanka) [inline yaptide/routes/batch_routes.py]:
```suggestion
```

move that info to issue description

**Comment by grzanka** [comment]
To me it looks OK, but @Derstilon should check first if everything is there in place

File: yaptide/routes/batch_routes.py (hunk 1 above)
Comment (p1003) [reply yaptide/routes/batch_routes.py:40]: Ok

Comment (p1003) [reply yaptide/routes/batch_routes.py]: https://github.com/yaptide/yaptide/issues/242#issuecomment-1430542060
Wrote it down here, consider resolving conversation @grzanka

File: yaptide/routes/celery_routes.py [hunk 2]
```diff
@@ -16,7 +16,7 @@
                                   get_input_files, cancel_simulation)
 
 
-class SimulationRun(Resource):
+class JobsDirect(Resource):
     """Class responsible for SHIELD-HIT12A simulations running"""
```
Comment (Derstilon) [inline yaptide/routes/celery_routes.py]: update description if class is responsible only for simulations in celery directly on backend

**Review by Derstilon — CHANGES_REQUESTED** [review]

**Review by Derstilon — DISMISSED** [review]

**Review by Derstilon — DISMISSED** [review]

**Review by Derstilon — APPROVED** [review]

**Pull request merged**